openedx.yaml

```diff
@@ -14,4 +14,16 @@ oeps:
     oep-5:
         state: False
         reason: TODO - Implement for this application if appropriate
+    oep-7:
+        state: False
+        reason: TODO - Implement for this application if appropriate
+    oep-18:
+        state: False
+        reason: TODO - Implement for this application if appropriate
+    oep-30:
+        state: False
+        reason: TODO - Implement for this application if appropriate
```

Comment:
It is appropriate to implement for this repo, since it contains models that should be annotated to indicate that they don't contain PII (or maybe some of them do?).  Let's follow Feanil's suggestion and throw a ticket in our backlog for this.

Reply:
agreed. Ticket in the backlog and noted in the doc